homeassistant/components/light/wemo.py

```diff
@@ -140,3 +148,89 @@ def turn_off(self, **kwargs):
     def update(self):
         """Synchronize state with bridge."""
         self.update_lights(no_throttle=True)
+
+
+class WemoDimmer(Light):
+    """Representation of a WeMo dimmer."""
+
+    def __init__(self, device):
+        """Initialize the WeMo dimmer."""
+        self.wemo = device
+        self._brightness = None
+        self._state = None
+        # look up model name once as it incurs network traffic
+        self._model_name = self.wemo.model_name
```

Comment:
If this uses I/O move it to the `async_added_to_hass` coroutine and schedule it on the executor.